easybuild/easyconfigs/h/HDF5/HDF5-1.8.18-foss-2016b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'HDF5'
+version = '1.8.18'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/current18/src/']
```

Comment:
@jhein please use `%(version_major)s%(version_minor)s` rather than hardcoding `18`?

We should also include `https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(versions)/src/` too, since the sources are probably going to move there if there's another `1.8.x` release...